main.go

```diff
@@ -37,5 +38,7 @@ func main() {
 		cmd.CmdAdmin,
 	}
 	app.Flags = append(app.Flags, []cli.Flag{}...)
-	app.Run(os.Args)
+	err := app.Run(os.Args)
```

Comment:
@tboerger Don't do that. Last time, it results in a bug I spent many time to fix it.